service_discovery_manager/Tools/weightFinder/browser/runAutomationTool.py

```diff
@@ -0,0 +1,88 @@
+#
+# Tool to run Automation testing.
+# Save the result as csv
+
+import sys, csv, socket, json
+from subprocess import check_output
+
+global g_ip
+global g_port
+
+global g_socket
+
+g_loadingTimeExtractor = './loadingtime_extract.js'
+g_availableFreq = [3200, 2500, 1800, 1100, 800]
```

Comment:
g_availableFreq is not matched renderer's g_availableFreq. Is it OK?

Reply:
Yes it is intentional. This reduces number of tests which had been too many tests before.